### PR TITLE
Fix python hover help for parameters

### PR DESCRIPTION
**Pull request by darzu**
Fixes: https://github.com/microsoft/pxt-microbit/issues/3067

Before:
image

After:
image

### CODE REVIEW

File: pxtcompiler/emitter/service.ts [hunk 1]
```diff
@@ -306,15 +326,21 @@ namespace ts.pxtc {
                     }
                     if (minVal) options['min'] = { value: minVal };
                     if (maxVal) options['max'] = { value: maxVal };
+                    const pyTypeString = (p.type && emitPyTypeFromTypeNode(p.type))
+                        || (paramType && emitPyTypeFromTsType(paramType))
+                        || "unknown";
+                    const initializer = p.initializer ? p.initializer.getText() :
+                        getExplicitDefault(attributes, n) ||
+                        (p.questionToken ? "undefined" : undefined)
+                    if (n === "myNum") {
```
Comment (jwunderl) [inline pxtcompiler/emitter/service.ts:335]: Looks like a leftover log for tracing?

File: pxtpy/converter.ts [hunk 2]
```diff
@@ -2578,6 +2578,7 @@ namespace pxt.py {
         apis.forEach(addSym)
 
         if (syntaxInfo && infoNode) {
+            console.log("converter.ts 2581")
```
Comment (jwunderl) [inline pxtpy/converter.ts:2581]: console.debug?

File: pxtcompiler/emitter/service.ts (hunk 1 above)
Comment (darzu) [reply pxtcompiler/emitter/service.ts:335]: Yes, thank you!

**Comment by darzu** [comment]
> the word "type" doesn't mean anything to me anymore =.=

Ha, what do you mean? Too much usage of the word "type"?

**Comment by livcheerful** [comment]
> > the word "type" doesn't mean anything to me anymore =.=
> 
> Ha, what do you mean? Too much usage of the word "type"?

haha yeah I just read it a lot

**Pull request merged**